stackcopy.py

```
@@ -729,9 +732,10 @@ def main():
                             })
 
                 for job in pending_copy_jobs:
+                    success = False
                     try:
                         success = job['future'].result()
-                        if success:
+                        if success: # noqa: BLE001 - top-level CLI error boundary
```

Ruff (0.14.5)

Unused `noqa` directive (unused: `BLE001`)

Remove unused `noqa` directive

(RUF100)